src/utils.jl

```diff
@@ -359,3 +359,63 @@ function directional_pad(scene, limits, tickcoord_in_inputspace, ticklabel::Abst
 
     return padding_vec
 end
+
+############################################################
+#                                                          #
+#     Splitting Of LineString / coast lines                #
+#                                                          #
+############################################################
+
+#`LineSplitting` module defines a `split` method for vectors of `LineString` objects.
+#
+#This is needed to fix e.g. coast line displays when lon_0 is not 0 but cutting polygons at lon_0+-180.
+
+module LineSplitting
+
+	import GeoMakie.LineString
+	import GeoMakie.Observable
+	import Base.split
+	
+	function regroup(tmp::Vector)
+		coastlines_custom=LineString[]
+		println(typeof(coastlines_custom))
+		for ii in 1:length(tmp)
+			push!(coastlines_custom,tmp[ii][:]...)
+		end
+		coastlines_custom
+	end
+	
+	function split(tmp::Vector{<:LineString}, lon0::Real)
+		[split(a,lon0) for a in tmp]
+	end
+	
+	function split(tmp::LineString, lon0::Real)
+		lon0<0.0 ? lon1=lon0+180 : lon1=lon0-180 
+		np=length(tmp)
+		tmp2=fill(0,np)
+		for p in 1:np
+			tmp1=tmp[p]
+			tmp2[p]=maximum( [(tmp1[1][1]<=lon1)+2*(tmp1[2][1]>=lon1) , (tmp1[2][1]<=lon1)+2*(tmp1[1][1]>=lon1)] )
+		end
+		if sum(tmp2.==3)==0
+			[tmp]
+		else
+			jj=[0;findall(tmp2.==3)...;np+1]
+			[LineString(tmp[jj[ii]+1:jj[ii+1]-1]) for ii in 1:length(jj)-1]
+		end
+	end
+
+	split(tmp::Vector,dest::Observable) = tmp
```

Comment:
Should this `@lift(tmp, $dest)` instead?

Reply:
not sure. current code works

Reply:
should be fixed in 835686b596492e5dce073e3289e2789827b3a18e